test_output/help/image_list.txt

```diff
@@ -8,7 +8,6 @@ knife bmcs image list (options)
         --chef-zero-port PORT        Port (or port range) to start chef-zero on.  Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works.
     -k, --key KEY                    API Client Key
         --[no-]color                 Use colored output, defaults to enabled
-        --compartment-id COMPARTMENT The OCID of the compartment.
```

Comment:
I think there was a miscommunication here: I mentioned removing compartment-id from common_options since it is no longer used in every operation, but we still want it to be an option in every command other than compartment list. While it should be common for users to work only in a single compartment, it should also be pretty common to work in more than one.